content/docs/getting-started.md

```diff
@@ -34,6 +34,7 @@ The daemon also uses a configuration file located in `/etc/containerd/config.tom
 A sample configuration file looks like this:
 
 ```toml
+version = 2
```

Comment:
I think we need to change `plugins.linux` section as well. Please try launching contained with the updated config.

Reply:
@kzys , Normally version field is at root level. But in case of "containerd", do we really need to specify the version explicitly for Plugins as well ?  Meanwhile, I'll try launching containerd with version field just at root level in the config file to see it's behavior.